mainchain-follower/db-sync-follower/src/candidates/mod.rs

```diff
@@ -1,14 +1,16 @@
-use crate::candidates::datum::RegisterValidatorDatum;
 use crate::db_model::{
 	self, Address, Asset, BlockNumber, EpochNumber, MainchainTxOutput, StakePoolEntry,
 };
 use crate::metrics::McFollowerMetrics;
 use crate::observed_async_trait;
 use crate::DataSourceError::*;
 use authority_selection_inherents::authority_selection_inputs::*;
-use datum::{DParamDatum, PermissionedCandidateDatums};
+use datum::raw_permissioned_candidate_data_vec_from;
+use datum::DParamDatum;
```

Comment:
Shouldn't DParam plutus data format be in the same place were permissioned and registered candidates?

Reply:
it is, that's some left-over public import that it got imported through. fixed that